### ISSUE TITLE
Toggle "switch" button needs to be finished and accessible

### ISSUE THREAD

**Issue by developpeurweb** [SAP/cloud-commerce-spartacus-storefront#2216]
- Currently doesn't receive any focus and can't be toggled with the keyboard.
- This may have some i18n issues for long worded languages, check with designers.
- Most likely to be tackled after the a11y process is in motion.

<img width="276" alt="Screen Shot 2019-04-29 at 1 36 28 PM" src="https://user-images.githubusercontent.com/15113219/56915173-da76f580-6a83-11e9-895f-e0120085451a.png">

Related to this tickets are:

- GH-2285
- PR #2103

**Comment by kacperknapik** [comment]
need more info from @developpeurweb

**Comment by developpeurweb** [comment]
Accessible Toggle Buttons were achieved on ticket GH-4398 so this ticket is obsolete... although Toggle Buttons will be most likely descoped as per @Xymmer request, in the meantime I'm removing this from A11Y milestones and team boards, and assigning to Bill for proper placement.
